flexget/plugins/input/smart_imdb.py

```diff
@@ -0,0 +1,478 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+import re
+from jsonschema.compat import str_types
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+from imdb import IMDb
```

Comment:
No idea. I'm not quite sure what the plugin does internally and I don't have time to taker a more thorough look at it :stuck_out_tongue: